src/it/hosting-maven_release_plugin_version-ok/invoker.properties

```diff
@@ -0,0 +1,21 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I just checked: those files were imported by @stephenc 
I think this might actually not be an issue since those (test) files are not packaged in the final "production" artifact. 
But surely Stephen will be more knowledgeable about licensing than me.